libparsec/crates/protocol/schema/authenticated_cmds/shamir_create_shared_recovery_device.json5

```diff
@@ -0,0 +1,71 @@
+[
+    {
+        "major_versions": [
+            4
+        ],
+        "req": {
+            "cmd": "shamir_recovery_setup",
+            "fields": [
+                {
+                    "name": "setup",
+                    // Set to `None` to clear previous Shamir recovery setup
+                    "type": "RequiredOption<ShamirRecoverySetup>"
+                }
+            ]
+        },
+        "reps": [
+            {
+                "status": "ok"
+            },
+            {
+                // Certificate is not signed by the authenticated user, or timestamp is invalid
+                "status": "invalid_certification"
+            },
+            {
+                // Cannot deserialize data into the expected certificate, or inconsistency
+                // between  certificates and/or threshold
+                "status": "invalid_data"
```

Comment:
Why not expose specific status? i.e. `invalid_recipient`, `recipient_not_in_brief`, `recipient_already_has_share`, etc.
It seems that this is how it was defined in the RFC, but I ask just out of curiosity.
cc @vxgmichel

Reply:
The RFC doesn't provide more specific status as far as I can tell.

More status could be useful for debug, although I would not expect the client code to behave differently depending on those status (it's probably a bug if the client sent invalid data).

So I don't really have an opinion about it.